$OpenBSD: patch-src_pulsecore_macro_h,v 1.1 2016/02/09 16:20:15 visa Exp $

On mips64, macro PAGE_SIZE is not really usable in userspace because
page size varies between systems.

--- src/pulsecore/macro.h.orig	Fri Feb  5 06:59:36 2016
+++ src/pulsecore/macro.h	Fri Feb  5 07:01:05 2016
@@ -44,6 +44,10 @@
 #endif
 #endif
 
+#if defined(__OpenBSD__) && defined(__mips__)
+#undef PAGE_SIZE
+#endif
+
 #if defined(PAGE_SIZE)
 #define PA_PAGE_SIZE ((size_t) PAGE_SIZE)
 #elif defined(PAGESIZE)
